pkg/controller/plan/kubevirt.go

```diff
@@ -54,8 +54,6 @@ const (
 	AnnOriginalName = "original-name"
 	//  Original VM name on source (value=vmOriginalID)
 	AnnOriginalID = "original-ID"
-	// DV deletion on completion
-	AnnDeleteAfterCompletion = "cdi.kubevirt.io/storage.deleteAfterCompletion"
```

Comment:
we create a DV
until now it was owned both by the DV and VM, now only the VM.
DV is removed once the import completes (by CDI garbage collector).
the importer pod owned by the PVC.

i am not sure about what should we care about when VM being cleaned..?

Reply:
in the last part I meant:
and then we don't care about the DV being cleaned when the importer pod is done?

Reply:
yes, but from what i've seen, even if the DV is in the PVC ownership, this will work.

Reply:
> yes, but from what i've seen, even if the DV is in the PVC ownership, this will work.

that's surprising, I'd expect the PVC to be removed when DV is removed when the DV is its sole owner but as we shouldn't have this situation anymore, it's irrelevant
ok so I think I got it now